mailbox/event/event-rabbitmq/src/test/java/org/apache/james/mailbox/events/RabbitMQEventBusTest.java

```diff
@@ -579,8 +580,7 @@ void registrationsShouldNotHandleEventsAfterStop() throws Exception {
 
                     TimeUnit.SECONDS.sleep(1);
                     assertThat(listener.numberOfEventCalls())
-                        .isEqualTo(callsAfterStop)
-                        .isLessThanOrEqualTo(MAX_EVENT_DISPATCHED_COUNT);
+                        .isCloseTo(callsAfterStop, Percentage.withPercentage(2));
```

Comment:
I don't get how it matches the test name: isCloseTo means +/- 2% while the test name tells that it should be inferior to some value.